src/mlpack/tests/q_learning_test.cpp

```diff
@@ -59,16 +59,16 @@ BOOST_AUTO_TEST_CASE(CartPoleWithDQN)
     double episodeReturn = agent.Episode();
     averageReturn(episodeReturn);
     /**
-     * Reaching running average return 60 is enough to show it works.
+     * Reaching running average return 35 is enough to show it works.
```

Comment:
Looks like a nice runtime/testtime improvement, thanks for looking into the issue.